sign_node/signer.py

```diff
@@ -111,11 +111,19 @@ def on_sync_request(self, queue, message):
             password = self.__password_db.get_password(
                 payload['key_id']
             )
+            sig_type = payload.get('sig_type', 'detach-sign')
```

Comment:
i'm not sure, but maybe we need to add validation for payload or put into `pydantic` schema

Reply:
We have validation on the build server, so no need to validate here